packages/asset-buyer/src/swap_quoter.ts

```diff
@@ -130,20 +186,20 @@ export class SwapQuoter {
      *
      * @return  An object that conforms to SwapQuote that satisfies the request. See type definition for more information.
      */
-    public async getSwapQuoteAsync(
+    public async getMarketBuySwapQuoteAsync(
         makerAssetData: string,
         takerAssetData: string,
-        makerAssetSwapAmount: BigNumber,
+        makerAssetBuyAmount: BigNumber,
         options: Partial<SwapQuoteRequestOpts> = {},
-    ): Promise<SwapQuote> {
+    ): Promise<MarketBuySwapQuote> {
```

Comment:
Can we de-dupe `getMarketBuySwapQuoteAsync` and `getMarketSellSwapQuoteAsync` as well?